Rocket.Chat/API/Requests/User/UpdateUserRequest.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  UserUpdateRequest.swift
+//  Rocket.Chat
+//
+//  Created by Filipe Alvarenga on 27/02/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+typealias UserUpdateResult = APIResult<UpdateUserRequest>
+
+class UpdateUserRequest: APIRequest {
+    let method: HTTPMethod = .post
+    let path = "/api/v1/users.update"
+
+    let userId: String
+    let user: User
+    let password: String?
+
+    init(userId: String, user: User, password: String? = nil) {
+        self.userId = userId
+        self.user = user
+        self.password = password
+    }
+
+    func body() -> Data? {
+        guard let name = user.name,
+                let username = user.username,
+                let email = user.emails.first else {
+            return nil
+        }
```

Comment:
We're using another style for guard... what do you think?

```swift
guard
    let name = user.name,
    let username = user.username,
    let email = user.emails.first
else {
    return nil
}
```

Reply:
It's really better. I'll change it.